clusterloader2/cmd/clusterloader.go

```diff
@@ -47,13 +50,15 @@ func validateClusterFlags() []error {
 
 func initFlags() {
 	flag.StringVar(&clusterLoaderConfig.ReportDir, "report-dir", "", "Path to the directory where the reports should be saved. Default is empty, which cause reports being written to standard output.")
-	flag.StringVar(&clusterLoaderConfig.TestConfigPath, "testconfig", "", "Path to the test config file")
+	pflag.CommandLine = pflag.NewFlagSet(os.Args[0], pflag.ContinueOnError)
+	pflag.StringArrayVar(&testConfigPaths, "testconfig", []string{}, "Paths to the test config files")
 	initClusterFlags()
```

Comment:
Can you move this line to the top? (keep all pflag commands in one block)

Reply:
I've change flag declaration to `pflag` only. However, I still need to keep `flag` due to `glog` adding it's standard flags only to `flag`.